sdk/aqueduct/models/operators.py

```diff
@@ -90,8 +89,15 @@ class MongoExtractParams(BaseModel):
 
 class ExtractSpec(BaseModel):
     service: ServiceType
-    integration_id: uuid.UUID
-    parameters: UnionExtractParams
+
+    # TODO(ENG-2994): This spec is parsed into a golang struct that still expects
+    #  the "integration" terminology.
+    resource_id: uuid.UUID = Field(alias="integration_id")
+    parameters: Union[str, UnionExtractParams]
+
+    class Config:
```

Comment:
cc @kenxu95 confirming these are for fields preserved in database?

Reply:
correct!